### PR TITLE
add init_stop() and fix descriptions

### PR DESCRIPTION
**Pull request by cgzones**
required by dpkg

### CODE REVIEW

File: policy/modules/system/init.if [hunk 1]
```diff
@@ -863,7 +863,7 @@ interface(`init_disable',`
 
 ########################################
 ## <summary>
-##	Reload all services (systemd).
+##	Reload the system (systemd).
```
Comment (pebenito) [inline policy/modules/system/init.if]: I don't think these summary changes are correct.

Comment (cgzones) [inline policy/modules/system/init.if]: I think the are:
https://github.com/systemd/systemd/blob/master/src/core/selinux-access.c#L228:
When there is a path (to a unit file) the class `service` is chosen, else the class `system` is chosen.

Comment (pebenito) [inline policy/modules/system/init.if]: Exactly. If you don't specify a unit, it will reload all of them.

**Comment by cgzones** [comment]
My bad

**Pull request closed without merging**